test/feeds.test.js

```diff
@@ -101,24 +101,90 @@ describe('test POST /feeds endpoint', () => {
     expect(feed.url).toEqual(feedData.url);
   });
 
-  it('no author being sent', async () => {
+  it('no author being sent, author is null or empty string', async () => {
     const feedData = {
       author: null,
       user: 'user',
       url: 'http://telescope200.cdot.systems',
     };
-    const res = await request(app).post('/feeds').send(feedData).set('Accept', 'application/json');
+    let res = await request(app).post('/feeds').send(feedData).set('Accept', 'application/json');
+    expect(res.status).toEqual(400);
+
+    feedData.author = '';
+    res = await request(app).post('/feeds').send(feedData).set('Accept', 'application/json');
     expect(res.status).toEqual(400);
   });
 
-  it('no url being sent', async () => {
+  it('no url being sent, url is null or empty string', async () => {
     const feedData = {
       author: 'foo',
       user: 'user',
       url: null,
     };
-    const res = await request(app).post('/feeds').send(feedData).set('Accept', 'application/json');
+    let res = await request(app).post('/feeds').send(feedData).set('Accept', 'application/json');
+    expect(res.status).toEqual(400);
+
+    feedData.url = '';
+    res = await request(app).post('/feeds').send(feedData).set('Accept', 'application/json');
+    expect(res.status).toEqual(400);
+  });
+
+  it('returnS 400 if url is invalid', async () => {
```

Comment:
Typo on `returnS`

Reply:
Omg! Silly me! I fix it when Im at my computer shortly.